### PR TITLE
Add show/hide code inside the index.html (docs)

### LINKED ISSUE #269: Add sample code to the demo page
I want to sample code on demo page.

**Is your feature request related to a problem? Please describe.**
It's hard to read HTML and CSS to use NES.CSS.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

127 0 0 1_5500_index html

### PR DESCRIPTION
**Pull request by guastallaigor**
ref #269 

**Description**
This adds a show/hide toggle button on top of each container in the `index.html` (docs).
When it clicks on show it displays a `<div>` in the bottom of the respective container, showing the code.
Inside this `<div>`, there's a "Copy" button, to copy the code to clipboard.
When you click the "Copy" button, it displays a balloon that says "Copied!".

**Compatibility**
N/A

**Caveats**
There's a lot of things that could be better in a near future for this.
This is just to have and easier access for the user to see/copy the code of each component, without having to use `devtools` or anything like that.
But I think we can close #269 if/when this is merged, and later open another issue for improvements.
Tested on Firefox and Chrome in Ubuntu 18.04 and added a few responsive adjustments.

**Demo**
screencast-localhost-4000-2019 02 06-23-12-11

### CODE REVIEW

**Comment by BcRikko** [comment]
Great 🎉 

But it's too complicated and it's too difficult to maintenance `index.html`. 😭 
So I think that I'll rebuild index.html and so on  from the ground up.

But I'm busy with bug fix, so please wait a moment. 🙏

**Comment by guastallaigor** [comment]
> It's so coool 
> 
> But it's too complicated and it's too difficult to maintenance `index.html`. 
> So I think that I'll rebuild index.html and so on from the ground up.
> 
> But I'm busy with bug fix, so please wait a moment.

I agree @BcRikko, but the reason here is to have something to address the lack of code/copy for better UX in 1.1.0.

In my opinion this could be accepted (or changed some things, I don't know), and in 1.2.0 can be rebuilded for better maintenance and a better code in general the whole `index.html` file, because the idea is to release 1.1.0 ASAP, and have a great code for maintenance takes a lot more time that I unfortunately don't have at the moment.
That's the best I could do with the time that I have :smile:

**Comment by BcRikko** [comment]
Oh, I see 👌 
Just as you said 👍

File: index.html [hunk 1]
```diff
@@ -44,19 +44,32 @@ <h1><i class="snes-jp-logo brand"></i>NES.css</h1>
...
         <button type="button" class="nes-btn is-success">Success</button>
         <button type="button" class="nes-btn is-warning">Warning</button>
         <button type="button" class="nes-btn is-error">Error</button>
         <button type="button" class="nes-btn is-disabled">Disabled</button>
       </div>
+      <button type="button" class="nes-btn is-error show-code" id="show-buttons-code"><></button>
+      <span class="nes-balloon from-right copied" id="copied-buttons">Copied!</span>
+    </section>
+    <section class="nes-container code" id="code-buttons">
+      <code>
```
Comment (BcRikko) [inline index.html]: The Preformatted Text element is better here ☝️ 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre
```html
<pre><code>
...
</code></pre>
```

File: index.html [hunk 2]
```diff
@@ -44,19 +44,32 @@ <h1><i class="snes-jp-logo brand"></i>NES.css</h1>
...
+        <p><span><</span>button type="button" class="nes-btn">Normal<span><</span>/button></p>
```
Comment (BcRikko) [inline index.html]: Is this correct HTML? 🤔 
I think it is better to use encoded code. 💻 

```html
<button type="button" class="nes-btn">Normal</button>
```
↓↓↓↓
```html
&lt;button type=&quot;button&quot; class=&quot;nes-btn&quot;&gt;Normal&lt;/button&gt;
```

File: index.html [hunk 3]
```diff
@@ -44,19 +44,32 @@ <h1><i class="snes-jp-logo brand"></i>NES.css</h1>
...
-      <div>
+      <div id="buttons">
         <button type="button" class="nes-btn">Normal</button>
         <button type="button" class="nes-btn is-primary">Primary</button>
         <button type="button" class="nes-btn is-success">Success</button>
         <button type="button" class="nes-btn is-warning">Warning</button>
         <button type="button" class="nes-btn is-error">Error</button>
         <button type="button" class="nes-btn is-disabled">Disabled</button>
       </div>
+      <button type="button" class="nes-btn is-error show-code" id="show-buttons-code"><></button>
```
Comment (BcRikko) [inline index.html]:
```suggestion
      <button type="button" class="nes-btn is-error show-code" id="show-buttons-code">&lt;&gt;</button>
```

File: index.html (hunk 2 above)
Comment (guastallaigor) [reply index.html]: The `&quot;` is going to take me more time to do it, i can't figure out a good regex to do it, so I think I will have to change it all manually :smile:

Comment (BcRikko) [reply index.html]: You can easily encode using the following tools: 🛠 

* https://www.freeformatter.com/html-escape.html
* https://webtools.dounokouno.com/htmlescape/
* https://codebeautify.org/html-escape-unescape
* etc...

File: index.html (hunk 1 above)
Comment (guastallaigor) [reply index.html]: Okay :+1: 
But without this:

```css
pre {
  white-space: normal;
}
```

Looks weird, too spaced, ok?

File: index.html (hunk 2 above)
Comment (guastallaigor) [reply index.html]: Thank you for this.

**Comment by BcRikko** [comment]
@Kraven1 What are you trying to say?  🤔

**Comment by BcRikko** [comment]
@guastallaigor  I fixed some codes. 🛠 
Could you review these? 🙏

**Comment by guastallaigor** [comment]
> @guastallaigor I fixed some codes. 
> Could you review these?

Looking great with `highlight.js` and the right html syntax display! Thank you for this.
I can't post a review in my on PR so I guess the comment is enough? :smile: 

I've test it and I hope you don't mind but I fixed two alignment tags in the `form` and `table` component in the syntax display.
But everything now is LGTM :+1: :rocket:

**Pull request merged**